.github/ISSUE_TEMPLATE/submit-method-package.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Submit method package
+about: Form to submit a new or updated method package
+
+---
+
+Thank you for submitting a method package to the HCA DCP Methods Registry!
+
+To expedite your package's addition to https://data.humancellatlas.org/analyze/methods,
+please provide the following package metadata.
+
+Required:
+- Tool title: 
+- Contact name: 
+- Contact email: 
+- Who to attribute: 
+- Language: 
+- GitHub repository URL: (_e.g. https://github.com/yourusername/yourpackagename_)
```

Comment:
Code repository URL: instead of "Github" - groups should be free to use Bitbucket, SourceForge or other source code hosting solutions...

Reply:
Done.